pkg/alertmanager/api_grafana_test.go

```diff
@@ -36,6 +36,7 @@ const (
 				"http_config": {
 					"enable_http2": true,
 					"follow_redirects": true,
+					"http_headers": null,
```

Comment:
Note to reviewers: this is the default value. Added so that the check done by `TestMultitenantAlertmanager_GetUserGrafanaConfig` pass.